libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -74,4 +76,36 @@ export class StripeManager {
       customer.tax?.automatic_tax === 'not_collecting'
     );
   }
+
+  /**
+   * If customer taxId does not match incoming taxId, update customer record with incoming taxId.
+   *
+   * @param customerId
+   * @param taxId
+   * @returns
```

Comment:
All params should have descriptions or should be omitted.

Reply:
Will update!

Looking at the other methods in this file and `stripe.client.ts`, I noticed not all of them are filled out or even filled out in a similar fashion. I didn't see anything explicitly mentioned in their PRs (all were merged recently).

Is there a reason they have been left blank (e.g. another ticket would take care of that or maybe an extension that auto-populates empty params/returns)? Otherwise, would you like me to update them as well or remove them?

Methods and PRs:
- `fetchActiveCustomer`, `finalizeInvoiceWithoutAutoAdvance`, `getMinimumAmount` and `finalizeInvoice` #16545
- `getSubscriptions` and `fetchSubscriptions` #16552
- `isCustomerStripeTaxEligible` #16465